packages/grafana-ui/src/components/Combobox/Combobox.tsx

```diff
@@ -135,12 +135,10 @@ export const Combobox = <T extends string | number>(props: ComboboxProps<T>) =>
 
         if (!optionMatchingInput) {
           const customValueOption = {
-            label: t('combobox.custom-value.label', 'Custom value: ') + inputValue,
```

Comment:
We do a check for this in `itemToString`, which we can remove now

Reply:
Do you mean removing [these lines](https://github.com/grafana/grafana/blob/eledobleefe/multicombobox-custom-value-96110/packages/grafana-ui/src/components/Combobox/filter.ts#L23-L25), right?